extractors/python_functions/org_extraction/code_snippet.md

```diff
@@ -1,11 +1,8 @@
 ```python
-from typing import List, Tuple
-
 YOUR_ATTRIBUTE: str = "text" # only text attributes
 YOUR_LABEL: str = "ORG"
 
-def org_extraction(record) -> List[Tuple[str, int, int]]:
-    
+def org_extraction(record):
```

Comment:
extractor

Reply:
See other comments about the extraction naming.